client/app/views/importer.coffee

```diff
@@ -10,16 +10,18 @@ module.exports = class ImporterView extends BaseView
     tagName: 'div'
     className: 'modal'
 
-    events: ->
-        'change #vcfupload': 'onupload'
-        'click  #confirm-btn': 'addcontacts'
-
 
     afterRender: ->
         @$el.modal()
         @upload = @$('#vcfupload')[0]
         @content = @$('.modal-body')
         @confirmBtn = @$('#confirm-btn')
+        # the events doesn't seem to be fired by Backbone when the application
+        # runs inside an iframe, so we need tu add them by hand
+        @$('#vcfupload').change () => @onupload()
```

Comment:
If you bind the events manually, you also need to unbind then when the view is destroyed.